report_qweb_decimal_place/reports/price_unit_value_format.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="price_unit_value_format">
+        <span
+            class="text-nowrap"
+            t-esc="price_unit"
+            t-options='{"widget": "float", "precision": currency.price_decimal_places}'
+        />
```

Comment:
```suggestion
        <t t-if="currency.apply_price_decimal_place">
            <span
                class="text-nowrap"
                t-esc="price_unit"
                t-options='{"widget": "float", "precision": currency.price_decimal_places}'
            />
        </t>
```
As we talked earlier, to minimize the code in the extension of individual reports.